apps/src/regionalPartnerMiniContact/regionalPartnerMiniContact.js

```diff
@@ -55,8 +55,9 @@ window.showRegionalPartnerMiniContactPopupLink = function() {
     <RegionalPartnerMiniContactPopupLink
       notes={notes}
       sourcePageId={sourcePageId}
-      children={linkText}
-    />,
+    >
+      {linkText}
+    </RegionalPartnerMiniContactPopupLink>,
```

Comment:
Intentional part of this change?

Reply:
Yes - see [related comment](https://github.com/code-dot-org/code-dot-org/pull/27825/files#r271476944).

Reply:
👍